packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -273,8 +334,34 @@ install kernel-open/nvidia-peermem.ko %{buildroot}%{_cross_datadir}/nvidia/open-
 
 # drm
 install kernel-open/nvidia-drm.ko %{buildroot}%{_cross_datadir}/nvidia/open-gpu/drivers/
+popd
```

Comment:
You are already at `NVIDIA-Linux-%{_cross_arch}-%{tesla_ver}`, which is what you used in line 342, why going out and in again if you can remain there and install if `x86_64`.